portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -35,27 +35,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     UserService userService = UserServiceFactory.getUserService();
     Gson gson = new Gson();
-    
+    String json = "";
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/";
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      String logoutUrl = userService.createLogoutURL("/");
 
       login_info.addToLoginList(userEmail);
       login_info.addToLoginList(logoutUrl);
 
       response.setContentType("application/json");
-      String json = gson.toJson(login_info);      
+      json = gson.toJson(login_info);      
     } 
     else {
-      String urlToRedirectToAfterUserLogsIn = "/";
-      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+      String loginUrl = userService.createLoginURL("/");
     
       login_info.addToLoginList("Guest");
       login_info.addToLoginList(loginUrl);
 
       response.setContentType("application/json");
-      String json = gson.toJson(login_info);
+      json = gson.toJson(login_info);
```

Comment:
Can't we just pull this out of the conditional?

Also, let's fix formatting to make newline for if/else consistent

```java
if () {
} else {
}

String json = gson.toJson(login_info);
```